dev/io.openliberty.jakartaee10.internal_fat/fat/src/io/openliberty/jakartaee10/internal/tests/EE10FeatureCompatibilityTest.java

```diff
@@ -474,8 +430,8 @@ public void transportSecurityUsed() {
         ee10FeaturesThatEnableSsl.add("connectorsInboundSecurity-2.0");
         ee10FeaturesThatEnableSsl.add("messagingSecurity-3.0");
 
-        for (String feature : features) {
-            if (nonEE10JavaEEFeatures.contains(feature) || nonEE10MicroProfileFeatures.contains(feature) || incompatibleValueAddFeatures.contains(feature)) {
+        for (String feature : allFeatures) {
+            if (!compatibleFeatures.contains(feature)) {
```

Comment:
Is there a reason you switched this from incompatibleFeatures.contains to !compatibleFeatures.contains?

Reply:
It's logically correct for what the test was doing (assuming I've understood it correctly).

`compatibleFeatures` are those that we want to assert are compatible
`incompatibleFeatures` are those that we want to assert are incompatible

This test is checking that features are compatible (i.e. that they don't cause conflicts), therefore it should only test features which are in the `compatibleFeatures` set.